docs/src/5-prestashop-billing/4-how-tos/4-handle-plan-selection/README.md

```diff
@@ -14,267 +14,273 @@ You should add your plan presentation in the configuration page for your module
 
 This is a simple working example that is purposefully basic, you can make the code as complex as you need it to be to cover your needs as long as you follow the steps in this guide.
 
-
 1. Create a JS file in your module: `views/js/configure.js` and inject it into `<module_name>.php` file.
```

Comment:
**issue**: ici il faut que tu explique plus la partie qui ajoute ce code et que tu l'extrait. Là c'est trop fat et le partenaire ne peut pas comprendre, donc il fera forcément du copier coller. Ce n'est pas ce qu'on veut.

De plus je pense que vaut mieux mettre le `$this->context->smarty->assign('componentItems' => $componentItems); ` directement après le code, pareil c'est plus simple de faire le lien que ça va ensemble plutôt que tout mettre à la fin.

Je pense que tu peux ajouter un numéro dans la liste, genre "2. Retrieve plan pricing from Billing API and inject it into your template"

Et mettre ce code :

```php
public function getContent()

  // ...

  /**********************
   * PrestaShop Billing *
   * *******************/

  $componentItems = [];
    // Load service for PrestaShop Billing
    $billingService = $this->getService('<module_name>.ps_billings_service');
    $productComponents = $billingService->getProductComponents();
    if (!empty($productComponents['body']['items']))
    {
        $componentItems = $productComponents['body']['items'];
    }
  // Allow to use $componentItems in your tpl file
  $this->context->smarty->assign('componentItems' => $componentItems);
  
   // ...

}
```

À mon sens ça n'est pas utile de mettre la gestion d'erreurs, c'est pas ce qu'on essaye d'expliquer. On veut que la personne comprenne comment utiliser ce qu'on lui propose, après il se débrouille sur sa gestion d'erreur.